doc/sphinxman/source/occ.rst

```diff
@@ -261,6 +266,12 @@ through "type select" values in the rightmost Table column.
     +                         +--------------------------------------------------------------+----------------------+----------------------+---------------------------+
     |                         | Cholesky-Decomposed Orbital-Optimized LCCD                   | RHF/UHF/ROHF/RKS/UKS | ---                  | |globals__cc_type| CD     |
     +-------------------------+--------------------------------------------------------------+----------------------+----------------------+---------------------------+
+    | occd                    | Orbital-Optimized CCD                                        | ---                  | ----                 | |globals__cc_type| CONV   |
```

Comment:
Done in 66b75fe5bca95ecf7cb69ddf639c3931ff1d209c.
I added it at the bottom of the tables, but given that @bozkaya also added stuff in the mean time, this will for sure create merge conflicts...